### ISSUE TITLE
Doesn't work with VS code on Linux Mint (x86-64)

### ISSUE THREAD

**Issue by egoholic** [elbywan/crystalline#90]
It just doesn't work (none of the features). Both master and 0.13.1.

I had an issue with compilation related to LLVM, which I fixed with recommendations from https://github.com/crystal-lang/crystal/issues/12896 

crystalline and crystal binaries placed withing /usr/bin/. Crystal plagin for VS Code has absolute path on crystalline.

VS Code 1.88.0

Extension status bar shows: ```Crystal: running context tool...``` and that is all. Sometimes when I hover some token the tooltip shows ```Loading....``` but most of the time really nothing happens.  

```
crystal version
Crystal 1.12.1 [4cea10199] (2024-04-11)

LLVM: 15.0.7
Default target: x86_64-unknown-linux-gnu
```

**Comment by egoholic** [comment]
Any suggestions?

**Comment by egoholic** [comment]
I kinda found a problem. My previous VS Code were installed from flatpak. When I installed VS Code from official site and installed and configured crystal extension for it then I found that Crystalline works well, so that the problem was not with crystalline, but with flatpak's VS Code build. I don't know what is different with flatpak's build, but I highly recommend to avoid it, because I found several more issues with it, including glitchy GUI.
